regulations/templates/regulations/node.html

```diff
@@ -14,6 +14,20 @@
 {% endif %}
 </p>
 {% endif %}
+
+<div class="comment-wrapper" data-section="{{ node.markup_id }}">
+  <div class="toggle">(click to toggle comment box)</div>
+  <div class="comment">
+    <h3>Enter your comment below:</h3>
+    <form>
+      <textarea role="textbox" aria-multiline="true"></textarea>
+      <div class="queued"></div>
+      <input type="file">
+      <button type="submit">Submit</button>
+    </form>
+  </div>
+</div>
```

Comment:
Not sure, but I think it makes sense for comments to apply to an entire section, not just intro text--we just need to make it clear visually that that's what's going on.

Do you imagine a custom `preamble.html` looking like this?

```
<li
    {% if node.marker %}marker-type="{{node.marker}}"{% endif %}
    {% if not inline %}id="{{node.markup_id}}" data-permalink-section{% endif %}
    {% if node.interp %}data-interp-id="{{node.interp.markup_id}}"{% endif %}
>
    {% include "regulations/node.html" %}
    <div class="comment">...</div>
</li>
```